pandas/tests/io/test_sql.py

```diff
@@ -741,6 +741,30 @@ def test_date_parsing(self):
             Timestamp("2010-12-12"),
         ]
 
+    @pytest.mark.parametrize(
+        "read_sql, text, mode",
+        [
+            (sql.read_sql, "SELECT * FROM types_test_data", ("sqlalchemy", "fallback")),
+            (sql.read_sql, "types_test_data", ("sqlalchemy")),
+            (
+                sql.read_sql_query,
+                "SELECT * FROM types_test_data",
+                ("sqlalchemy", "fallback"),
+            ),
+            (sql.read_sql_table, "types_test_data", ("sqlalchemy")),
+        ],
+    )
+    def test_custom_dateparsing_error(self, read_sql, text, mode):
+        if self.mode in mode:
```

Comment:
can you make this tests assert the results, otherwise its not actually testing anything

Reply:
Done